Library/Formula/pam-u2f.rb

```diff
@@ -0,0 +1,41 @@
+class PamU2f < Formula
+  desc "Provides an easy way to use U2F-compliant authenticators with PAM."
+  homepage "https://developers.yubico.com/pam-u2f/"
+  url "https://github.com/Yubico/pam-u2f.git", :commit => "95fb0ec65475e8afa15aef1c14d1457d1f80fece"
```

Comment:
I don't know if you've spotted it but there's a new upstream release [available](https://github.com/Yubico/pam-u2f/releases/tag/pam_u2f-1.0.2) which is newer than the commit you're using. Is that one problematic as well?